src/components/NormalLeftRow/index.jsx

```diff
@@ -144,12 +144,12 @@ function NormalLeftRow({ classes, treeNode, refType, setSchemaTree }) {
   const refIcon = {
     none: null,
     default: (
-      <IconButton aria-label="expand-ref" color="inherit">
+      <IconButton aria-label="expand-ref" color="inherit" size="small">
         <ExpandIcon fontSize="large" />
       </IconButton>
     ),
     expanded: (
-      <IconButton aria-label="shrink-ref" color="inherit">
+      <IconButton aria-label="shrink-ref" color="inherit" size="small">
         <ShrinkIcon fontSize="large" />
       </IconButton>
     ),
```

Comment:
With mdi-react, we have more control on the size:

```diff
diff --git a/package.json b/package.json
index 932ed6a..19b93c6 100644
--- a/package.json
+++ b/package.json
@@ -45,6 +45,7 @@
     "classnames": "^2.2.6",
     "markdown-it": "^10.0.0",
     "markdown-it-highlightjs": "^3.0.0",
+    "mdi-react": "^6.7.0",
     "prop-types": "^15.7.2",
     "ramda": "^0.26.1"
   }
diff --git a/src/components/NormalLeftRow/index.jsx b/src/components/NormalLeftRow/index.jsx
index 4d28ac2..3dc8c43 100644
--- a/src/components/NormalLeftRow/index.jsx
+++ b/src/components/NormalLeftRow/index.jsx
@@ -4,8 +4,8 @@ import { makeStyles } from '@material-ui/core/styles';
 import classNames from 'classnames';
 import Typography from '@material-ui/core/Typography';
 import IconButton from '@material-ui/core/IconButton';
-import ExpandIcon from '@material-ui/icons/ArrowRightRounded';
-import ShrinkIcon from '@material-ui/icons/ArrowDropDownRounded';
+import ExpandIcon from 'mdi-react/MenuRightIcon';
+import ShrinkIcon from 'mdi-react/MenuDownIcon';
 import WarningIcon from '@material-ui/icons/ReportProblemOutlined';
 import Tooltip from '../Tooltip';
 import { basicTreeNode, NOOP } from '../../utils/prop-types';
@@ -27,7 +27,7 @@ import {
  */
 const useStyles = makeStyles(theme => ({
   indentation: {
-    marginLeft: indent => `${theme.spacing(indent * 2)}px`,
+    marginLeft: indent => theme.spacing(indent * 2),
   },
 }));
 
@@ -151,12 +151,12 @@ function NormalLeftRow({
     none: null,
     default: (
       <IconButton aria-label="expand-ref" color="inherit" size="small">
-        <ExpandIcon fontSize="large" />
+        <ExpandIcon size={24} />
       </IconButton>
     ),
     expanded: (
       <IconButton aria-label="shrink-ref" color="inherit" size="small">
-        <ShrinkIcon fontSize="large" />
+        <ShrinkIcon size={24} />
       </IconButton>
     ),
   }[refType];
diff --git a/yarn.lock b/yarn.lock
index 494d68b..bee99c4 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -7838,6 +7838,11 @@ md5.js@^1.3.4:
     inherits "^2.0.1"
     safe-buffer "^5.1.2"
 
+mdi-react@^6.7.0:
+  version "6.7.0"
+  resolved "https://registry.yarnpkg.com/mdi-react/-/mdi-react-6.7.0.tgz#d291f919b5f0f99cbbeac111f2c9ca5184607b6b"
+  integrity sha512-Xpl3bXh4Nmuufoll8ejtErsDd0v/Ty+Je+qC9V3owriFUQZno91F/f0CkAdQj9fRqWY+LwjhFcWii+3lFFGKvQ==
+
 mdn-data@2.0.4:
   version "2.0.4"
   resolved "https://registry.yarnpkg.com/mdn-data/-/mdn-data-2.0.4.tgz#699b3c38ac6f1d728091a64650b65d388502fd5b"
```

#### Before
<img width="141" alt="Screen Shot 2020-02-25 at 4 50 00 PM" src="https://user-images.githubusercontent.com/3766511/75290715-e0964980-57ee-11ea-9f67-0c33798797d2.png">

#### After
<img width="104" alt="Screen Shot 2020-02-25 at 4 46 04 PM" src="https://user-images.githubusercontent.com/3766511/75290564-99a85400-57ee-11ea-9a12-5c3ec3751761.png">

Additionally, it would be nice to not have the hover touch the text on its left.

Reply:
This is great 👍 Applied changes!